packages/client/utils/constants.ts

```diff
@@ -92,6 +92,7 @@ export const NOTIFICATIONS = 'notifications'
 /* Org Settings */
 export const BILLING_PAGE = 'billing'
 export const MEMBERS_PAGE = 'members'
+export const ORG_SETTINGS_PAGE = 'org-settings'
```

Comment:
+1 because it's on a subpage of `organizations/:orgId/` the `org-` part is redundant and could be removed.